governance/polls/Adjust the WBTC Debt Ceiling - July 27, 2020

```diff
@@ -0,0 +1,35 @@
+---
+title: Adjust the WBTC Debt Ceiling - July 27, 2020
+summary: Signal your support for increasing the WBTC Debt Ceiling from 20MM to 34MM.
+discussion_link: https://forum.makerdao.com/t/signal-request-should-we-raise-wbtc-debt-ceiling/3223?u=andy_mccall
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Adjust the WBTC Debt Ceiling - July 27, 2020
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Maker Governance Community. The community can use this poll express support or opposition to increasing the WBTC Debt Ceiling from 20MM to 34MM.
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/governance/governance#is-there-more-than-one-type-of-vote)) will be active for three days beginning on Monday, July 27 at 5 PM UTC, the results of which may inform an Executive Vote ([FAQ](https://community-development.makerdao.com/governance/governance#what-is-continuous-approval-voting)) which will go live on Friday, July 31, at 5 PM UTC.
```

Comment:
It's 16:00 UTC, not 5PM.

Reply:
For both polls and execs (at least it's the target)

Reply:
👍